app/styles/utilities/_borders.less

```diff
@@ -17,7 +17,7 @@
   border-color: var(--color-border-default);
 }
 
-.border-color-alert {
+.border-color-warning {
```

Comment:
⚠️  Also super dangerous to update this like this

Reply:
this time I checked upfluence-wide !
this is the single occurence of this class